src/sdk/pynni/nni/curvefitting_assessor/curvefitting_assessor.py

```diff
@@ -69,21 +74,32 @@ def assess_trial(self, trial_job_id, trial_history):
         assess whether a trial should be early stop by curve fitting algorithm
         return AssessResult.Good or AssessResult.Bad
         '''
+        self.trial_job_id = trial_job_id
         self.trial_history = trial_history
         curr_step = len(trial_history)
         if curr_step < self.start_step:
             return AssessResult.Good
+        if (curr_step - self.start_step) / self.gap < self.judgment_num:
+            return AssessResult.Good
+        self.judgment_num = (curr_step - self.start_step) / self.gap
```

Comment:
is self.judgment_num a float? Then it almost never runs line 83

Reply:
Turn it into divisible